src/pages/en/guides/markdown-content.mdx

```diff
@@ -583,6 +575,58 @@ const { minutesRead } = Astro.props.frontmatter;
 </html>
 ```
 
+### Extending Markdown config from MDX
+
+By default, Astro's MDX integration will extend [your project's existing Markdown configuration](https://docs.astro.build/en/reference/configuration-reference/#markdown-options). To override individual options, you can specify their equivalent in your MDX configuration.
+
+For example, say you need to disable GitHub-Flavored Markdown and apply a different set of remark plugins for MDX files. You can apply these options like so, with [the `extendMarkdownConfig` option](/en/guides/integrations-guide/mdx/#extendmarkdownconfig) enabled by default:
+
+```ts
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  markdown: {
+    syntaxHighlight: 'prism',
+    remarkPlugins: [remarkPlugin1],
+    githubFlavoredMarkdown: true,
+  },
+  integrations: [
+    mdx({
+      // `syntaxHighlight` inherited from Markdown
+
+      // Markdown `remarkPlugins` ignored,
+      // only `remarkPlugin2` applied.
+      remarkPlugins: [remarkPlugin2],
+      // `githubFlavoredMarkdown` overridden to `false`
+      githubFlavoredMarkdown: false,
+    })
+  ]
+});
+```
+
+You may also need to disable `markdown` config extension in MDX. For this, set `extendMarkdownConfig` to `false`:
```

Comment:
```suggestion
You may also need to disable your `markdown` config settings in MDX. For this, set [the `extendMarkdownConfig` option (enabled by default)](/en/guides/integrations-guide/mdx/#extendmarkdownconfig) to `false`:
```

Reply:
I think "disable your Markdown config settings" is a bit confusing. Ex. does that disable syntax highlighting?